phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -152,6 +158,9 @@ public IndexRebuildRegionScanner(final RegionScanner innerScanner, final Region
             verifyType = IndexTool.IndexVerifyType.fromValue(valueBytes);
             if (verifyType != IndexTool.IndexVerifyType.NONE) {
                 verify = true;
+                boolean shouldLogBeyondMaxLookbackInvalidRows =
+                    config.getBoolean(PHOENIX_LOG_BEYOND_MAX_LOOKBACK_ERROR_ROWS_CONF_KEY,
```

Comment:
Can this property be modified via -D option from the client-side? Good to have a test for that.

Reply:
So far I have done nothing special to allow it be overridden from client-side. If the IndexTool is already taking all conf parameters from the client and using them on the server-side, then that's the case for this as well, and if it's not, it's not. (At a cursory look it appears that it's only certain flags which the InputFormat explicitly passes as scan attributes to the coproc?)

My original intention was that this would be a cluster-wide setting. I could change it to be overridable. 

@swaroopak - if this were overridable from the client-side, what kind of test would be sufficient? Given the long running time of our index tool tests already, I'm reluctant to add yet-another-end-to-end test just to check that a flag is carried through, but I'm not sure what other options there are to actually verify that each of the several layers passes the config through properly.

Reply:
It is important that we have this property overridable for each IndexTool run. You could modify the existing/the one added in this PR run of IndexTool to make sure the property changes from the client-side are reflecting properly.

Reply:
@swaroopak - Added a test in latest commit.